addons/mail_bot/views/discuss.xml

```diff
@@ -10,8 +10,17 @@
         @param {mail.NotificationAlert} widget
     -->
     <t t-name="mail.NotificationAlert">
-       <center t-if="widget.isNotificationBlocked" class="o_notification_alert alert alert-primary">
-           Odoo Push notifications have been blocked. Go to your browser settings to allow them.
-       </center>
+        <div class="o_notificationAlert alert alert-primary" t-if="widget.isNotificationBlocked">
+            Odoo Push notifications have been blocked. Go to your browser settings.
+            <div t-if="widget.settingUrl" class="o_notificationAlert_url">
+                <span class="border border-primary rounded-left p-2">
+                    <t t-esc="widget.settingUrl"/>
+                </span>
+                <button class="btn btn-sm btn-primary o_notificationAlert_urlClipboardButton">
+                    <span class="fa fa-clipboard"> Copy Link </span>
+                </button>
```

Comment:
classnames `o_notificationAlert*` should have the part `notificationAlert` in PascalCase:
```
o_NotificationAlert
o_NotificationAlert_url
o_NotificationAlert_urlClipboardButton
```